### PR TITLE
use file_line for bootstrap.cfg

### PR DESCRIPTION
**Pull request by mmoll**
@brandonweeks @alexjfisher @jyaworski please have a look if this works. If yes, I'll extend this after #372 went in to add that missing line for puppetserver 2.3.

### CODE REVIEW

**Comment by alexjfisher** [comment]
Looks good.  Works for me.

**Comment by brandonweeks** [comment]
:+1:

File: manifests/server/puppetserver.pp [hunk 1]
```diff
@@ -96,11 +95,26 @@
                 'set @simple[. = "puppetlabs.services.ca.certificate-authority-disabled-service"]/@value certificate-authority-disabled-service',],
   }
 
-  augeas { 'puppet::server::puppetserver::server_ca':
-    context => "/files${server_puppetserver_dir}/bootstrap.cfg",
-    changes => $augcmds,
```
Comment (domcleal) [inline manifests/server/puppetserver.pp]: Also delete the $augcmds variable in the lines above.

Comment (mmoll) [inline manifests/server/puppetserver.pp]: thanks, done.

**Comment by domcleal** [comment]
Did you mean to cancel the Travis CI build (https://travis-ci.org/theforeman/puppet-puppet/builds/122464474)?

**Comment by mmoll** [comment]
@domcleal wanted to cancel the build of that push in mmoll/puppet-puppet, thanks!

**Comment by domcleal** [comment]
Merged, thanks @mmoll and @alexjfisher, @brandonweeks for testing it.

**Pull request closed without merging**